libs/@blockprotocol/type-system/crate/Cargo.toml

```diff
@@ -32,3 +32,6 @@ wasm-bindgen-test = "0.3.37"
 lto = true
 opt-level = 's'
 panic = 'abort'
+
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = false
```

Comment:
Do you know why the AI opted out of wasm-opt?

Reply:
It was struggling to get it to build otherwise IIRC. Might be something we can look to re-enable in a follow-up.

@TimDiekmann Are we using this crate anywhere still?